ricecooker/chefs.py

```diff
@@ -281,10 +283,57 @@ def save_channel_tree_as_json(self, channel):
         self.CHEF_RUN_DATA['tree_archives']['current'] = filename.replace(os.getcwd() + '/', '')
         self.save_chef_data()
 
+    def save_channel_metadata_as_csv(self, channel):
+        # create data folder in chefdata
+        # config.LOGGER.info('SAVE METADATA TO CSV')
+        DATA_DIR = os.path.join('chefdata', 'data')
+        if not os.path.isdir(DATA_DIR):
+            os.makedirs(DATA_DIR)
```

Comment:
Not a big issue, but Python 3's `makedirs` can be called with `exist_ok=True` now, so that you don't have to check if the dir exists first. `exist_ok` was added in Python 3.2, so it's safe for any of our Python 3 only code to use. (We don't support Python 3 below 3.4 at this point.)

Reply:
Got it. Updated to use exist_ok = True